pkg/config/user_config.go

```diff
@@ -533,10 +534,11 @@ func GetDefaultConfig() *UserConfig {
 				BulkMenu: "b",
 			},
 		},
-		OS:                   GetPlatformDefaultConfig(),
-		DisableStartupPopups: false,
-		CustomCommands:       []CustomCommand(nil),
-		Services:             map[string]string(nil),
-		NotARepository:       "prompt",
+		OS:                           GetPlatformDefaultConfig(),
+		DisableStartupPopups:         false,
+		CustomCommands:               []CustomCommand(nil),
+		Services:                     map[string]string(nil),
+		NotARepository:               "prompt",
+		PromptToReturnFromSubprocess: true,
```

Comment:
Can we specify this default value in our config doc at docs/Config.md?

Reply:
updated config file with promptToReturnFromSubprocess default value and short desc